src/pages/office/office.module.ts

```diff
@@ -1,24 +1,9 @@
 import { NgModule } from '@angular/core';
-import { IonicPageModule } from 'ionic-angular';
-import { OfficePage } from './office';
-import { EffectsModule } from '@ngrx/effects';
-import { OfficeAnalyticsEffects } from './office.analytics.effects';
-import { ComponentsModule } from '../../components/common/common-components.module';
-import { OfficeComponentsModule } from './components/office.components.module';
-import { OfficeEffects } from './office.effects';
+import { OfficeCatBPageModule } from './cat-b/office.cat-b.page.module';
```

Comment:
I do see that other pages have a `*.module.ts` which basically imports the same page from different categories, but.. do we need this?

Reply:
Any thoughts on this? Was 3 days since the comment